extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -628,6 +637,10 @@ class QueueProcessor extends EventEmitter {
     }
 
     _getZkSiteNode() {
+        if (this.isReplayTopic) {
+            return `${zookeeperNamespace}${zkReplayStatePath}/${this.topic}/${this.site}`;
```

Comment:
Today, `${zookeeperNamespace}${zkReplayStatePath}/${this.topic}/${this.site}` and `${zookeeperNamespace}${zkStatePath}/${this.site}` zk path will share the same state. 
Stopping and pausing a site will pause both the site replication and the site replay work. 
Is it the expected behavior?

Reply:
We do keep separate states in zookeeper on purpose, to avoid race conditions (eg when updating the state);
but every instance will react to the same "commands" (eg pause/resume a site), so their states should be the same indeed.